includes/definitions/discovery/eatonpdu.yaml

```diff
@@ -21,6 +26,16 @@ modules:
                     index: 'outletCurrent.{{ $index }}'
                     high_limit: outletCurrentUpperCritical
                     warn_limit: outletCurrentUpperWarning
+                -
+                    oid: inputCurrent
+                    num_oid: '.1.3.6.1.4.1.534.6.6.7.3.3.1.4.{{ $index }}'
+                    index: '0.1.1'
+                    descr: 'Input current'
+                    divisor: 1000
+                    low_warn_limit: '.1.3.6.1.4.1.534.6.6.7.3.3.1.6.{{ $index }}'
```

Comment:
This syntax for the limits does not look right. You need to load define OID (the complete set of indexed variables) and value (the actual value you want to graph) and then all other variables of this index will be available. You can also pre-cache the values if they are part of another table/entry. 
That would allow you to define the limit using `low_warn_limit: '{{ $inputCurrentLowWarning }}'`(if we suppose inputCurrentLowWarning is the name associated with .1.3.6.1.4.1.534.6.6.7.3.3.1.6).